app/models/request.rb

```diff
@@ -89,27 +93,28 @@ def validator_for(request_option)
         ]
       end
 
-      select('min(uuids.external_id) AS group_id, GROUP_CONCAT(DISTINCT pw_location.description SEPARATOR ",") AS group_into, requests.*').
-      joins(joins + [
-        'INNER JOIN maps AS pw_location ON pw.map_id=pw_location.id',
+      select('min(uuids.external_id) AS group_id, GROUP_CONCAT(DISTINCT pw_location.description SEPARATOR ",") AS group_into, MIN(requests.id) AS id, MIN(requests.submission_id) AS submission_id, MIN(requests.request_type_id) AS request_type_id').
```

Comment:
This was in the previous code of Sequencescape, not in this branch.
About this: select('min(uuids.external_id) AS group_id
 I think after grouping the uuid is the same for all the group, and so min() will return the same uuid. However, if uuids.external_id is an UUID, are we looking for the oldest uuid?  Are UUIDs generated always in ascending order?